bin/run_test_suite_with_coverage

```diff
@@ -5,17 +5,22 @@ $rootDir = __DIR__ . '/..';
 $returnVal = 0;
 
 $env = getenv('ENV') ?: 'dev';
-$command = 'php -dzend_extension=xdebug.so vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover.xml';
+$command = 'php -dzend_extension=xdebug.so vendor/bin/phpunit';
+
+$command .= (isset($argv[1]) && $argv[1] == '--html')
+    ? ' --coverage-text --coverage-html=coverage-html'
```

Comment:
Why are we adding this `--html` output option? 
Should we make it `coverage.html`?

Reply:
It's a folder, so it should not be coverage.html, and when developing opening html files and browsing through it to see where you need to improve coverage is much easier than reading an xml file :)